src/util.js

```diff
@@ -0,0 +1,10 @@
+const myContainer = {};
```

Comment:
tiny hint: the name is ok for pet projects, but it is worth making it to have more sense for real projects